### DISCUSSION TITLE
Issue Reunning Debugger in Dev Container for VSCode

### DISCUSSION THREAD

**Discussion by LoopThrough-i-j** [microsoft/vscode-discussions#191] [Extension Development QnA] [answered]
Hi, I am facing an issue. Every time I use the VS Code debugger to run the source code in the dev container, I need to delete the `out` folder and rebuild the full thing, otherwise, the changes I made to the code are not getting reflected. I am surely missing something. Can anyone help?

**Comment by gjsjohnmurray** [comment, marked as answer]
I suggest you raise this issue on https://github.com/microsoft/vscode-remote-release instead. This discussion forum is currently focused on being a resource for people who are developing VS Code extensions. See the pinned topic at the top.
